### PR TITLE
Added function to get Graffic Buffer Info

### PR DESCRIPTION
**Pull request by munish-b**

### CODE REVIEW

File: os/android/utils_android.h [hunk 1]
```diff
@@ -118,6 +118,7 @@ static uint32_t DrmFormatToHALFormat(int format) {
     case DRM_FORMAT_XBGR8888:
     case DRM_FORMAT_RGBX8888:
     case DRM_FORMAT_ABGR8888:
+      return HAL_PIXEL_FORMAT_RGBA_8888;
```
Comment (tpalli) [inline os/android/utils_android.h]: this seems wrong, for all of these formats we return same format?

Comment (munish-b) [reply os/android/utils_android.h]: I will update. Thanks

**Comment by munish-b** [comment]
I will add public function to get Buffer Information.

**Comment by tpalli** [comment]
This sounds like API for gralloc (minigbm) and not hwcomposer (?) Which component will use the API?

**Comment by munish-b** [comment]
You are correct this API is needed to get information of Allocated Graphics Buffers.
Graphics buffers are being allocated using "new android::GraphicBuffer(width, height, format, usage)" and later buffer information is used to allocate additional buffers for internal validation purposes. I need to check if I can directly use CreateBuffer and ImportBuffer to avoid this change.

**Comment by kalyankondapally** [comment]
@tpalli This is for hwc-val test suite and we want to keep the gralloc/gbm backend transparent in the test suite.
@munish-b Ya, you want to check existing API, instead of adding new ones.

**Pull request closed without merging**